test/userController.test.js

```diff
@@ -94,3 +94,143 @@ describe('GET /api/v1/users/:user_id endpoint', () => {
     expect(response.status).toBe(404);
   });
 });
+
+describe('PATCH /api/v1/users/updateCurrentUser endpoint', () => {
+  it('should update the current user', async () => {
+    const signedCookie = await loginAndReturnCookie({
+      email: 'ava@ava.com',
+      password: 'secret',
+    });
```

Comment:
Since we're always logging in with the testUserCredentials that were defined on lines 13-16, we can simplify this a bit:

```suggestion
    // Arrange: Log in and get a signed cookie with valid test user credentials
    const signedCookie = await loginAndReturnCookie(testUserCredentials);
```